### DISCUSSION TITLE
Add a piece of documentation to i3 docs (Disabling title bar in tabbed / stacking layout)

### DISCUSSION THREAD

**Discussion by RayZ0rr** [i3/i3#4814] [Ideas]
Request to add information in [i3 docs](https://i3wm.org/docs/) about the age-old debate on optional disabling of _title bars_ in `stacking` / `tabbed` layout.

Easy answer from here : https://github.com/i3/i3/issues/4490
```sh
# Font to 1 or 0
font pango:Fira Nerd Font Mono 0

# Window borders to pixel 1
default_border pixel 1
default_floating_border pixel 1
```

Relevant discussion : https://github.com/i3/i3/issues/2664
(relevant answer? : https://github.com/i3/i3/issues/2664#issuecomment-632246008)
Relevant patch ( add warning '**Should be done locally. Use at your own risk**') : https://github.com/i3/i3/pull/2735

**Comment by Airblader** [comment]
Disabling title bars of stacked / tabbed containers is not something we (currently) support, and what you mention here are simply workarounds; workarounds do not belong in the i3 docs, and neither do external patches. Other than that, see [this answer](https://github.com/i3/i3/discussions/4813#discussioncomment-2048930).
